app/src/main/res/values/strings.xml

```diff
@@ -220,7 +222,7 @@
         <item quantity="one">There is %s program with specific settings</item>
         <item quantity="other">There are %s programs with specific settings</item>
     </plurals>
-    <string name="settings_sync_period">Syncing period</string>
+    <string name="settings_sync_period">Sync period</string>
```

Comment:
Same here? Not sure if this could affect the translation in other languages. But I guess we should do the same